DataAccess/DataAccess.Postgre/PostgreRepositories.cpp

```diff
@@ -219,7 +218,7 @@ std::uint64_t LoginRepository::loginUser(const Models::LoginInfo& loginInfo)
     catch (const std::exception& e)
     {
         Base::Logger::FileLogger::getInstance().log(
-            std::string(e.what() +'\n'),
+            std::string(std::string(e.what()) +'\n'),
```

Comment:
Why are we convert C-style string to std::string? Is it so necessary?

Reply:
Yeah, it is, because after that we add another one symbol(char). It could corrupt message

Reply:
OK